docs_src/data_block.ipynb

```diff
@@ -1023,6 +1023,186 @@
     "  - [`CollabList`](/collab.html#CollabList) for collaborative filtering"
```

Comment:
<p>How `ItemList.get` and `ImageList.get` differ</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2028/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='105'/>